### PR TITLE
Anomali ThreatStream integration

### PR DESCRIPTION
**Pull request by nitzanshapira**
Fixes https://github.com/demisto/server/issues/4705.
Including @dorsha fixes.

### CODE REVIEW

File: Integrations/integration-AnomaliThreatStream.yml [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+        'threatstream-intelligence': {
+            method: 'GET',
+            url: 'intelligence/',
+            version: 'v2',
+        },
+    }
+
+    var sendRequest = function(method, url, version, args, headers, multipart) {
+        url = base + version + '/' + url;
+        if (method === 'GET') {
```
Comment (Spivaka) [inline Integrations/integration-AnomaliThreatStream.yml]: we use anything else than get?
do we still user multipart?
I think not so the code shouldn't include those cases

File: Integrations/integration-AnomaliThreatStream.yml [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+            var cred = {
+                username: args.username,
+                api_key: args.api_key,
+            };
+            delete(args.username);
+            delete(args.api_key);
+            url += encodeToURLQuery(cred);
+        }
+
+        console.log(url);
```
Comment (Spivaka) [inline Integrations/integration-AnomaliThreatStream.yml]: remove

File: Integrations/integration-AnomaliThreatStream.yml [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+
+        var file_id = args.file_id;
```
Comment (Spivaka) [inline Integrations/integration-AnomaliThreatStream.yml]: Also can be removed

File: Integrations/integration-AnomaliThreatStream.yml [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+
+    var buildEmptyEntryContext = function() {
+        return {Type: entryTypes.note, Contents: null, ContentsFormat: formats.json, EntryContext: {}};
+    }
+
+    var isStandardCommand = function(command) {
+        return (command == 'ip' || command == 'file' || command == 'domain');
+    }
+
+    var convertKeysToPascalCase = function(dict) {
```
Comment (Spivaka) [inline Integrations/integration-AnomaliThreatStream.yml]: Shouldn't be a part of common?

**Comment by Spivaka** [comment]
fixes https://github.com/demisto/server/issues/4705

File: Integrations/integration-AnomaliThreatStream.yml (hunk 1 above)
Comment (nitzanshapira) [inline Integrations/integration-AnomaliThreatStream.yml]: At the moment we don't (I also removed the "import" functions which you implemented), but we may do so in the future. @NeelRust asked for the "upload file" feature (currently not implemented) which will require the "post" and "multipart" features.
I can remove it, but it works now and will probably be used in the future. Whatever you think.

**Pull request merged**